logger.go

```diff
@@ -83,11 +88,7 @@ func (s *SolveStatusLogger) tee() {
 		// Do internal redacting of secrets from any log output, to try and reduce the risk of accidental
 		// logging of secrets
 		for i := range status.Logs {
-			for _, value := range EnvVars {
-				if value.Secret && len(value.Value) > 0 {
-					status.Logs[i].Data = []byte(strings.ReplaceAll(string(status.Logs[i].Data), value.Value, "<redacted>"))
-				}
-			}
+			status.Logs[i].Data = []byte(redactor.redact(string(status.Logs[i].Data)))
```

Comment:
Probably should make redact just take []byte since this is the only use case instead of a bunch of casts here?

Reply:
Yes, sure - will update.